### DISCUSSION TITLE
Debugging

### DISCUSSION THREAD

**Discussion by HavardNJ** [shannah/jdeploy#112] [Q&A]
Hi, my exe does not work which probably means I forgot some jar or something. Is there any way of getting a log when running the exe? Tried running it from command line but got no output. I'm on windows.

**Comment by shannah** [comment]
On Windows currently it can be a bit tricky.

Best is probably to just find your app's jar file (it will be in the `$HOME/.jdeploy/packages/<your-app-id>/<your-app-version>/jdeploy-bundle`

Then run it with `java -jar your-app.jar`

You can use the version of java that is also in the .jdeploy directory,
